### PR TITLE
AP_Airspeed: move to per-instance params

### PR DESCRIPTION
**Pull request by IamPete1**
Moves to per-instance, renames first instance from `ARSPD_` to `ARSPD1_` and adds global enable parameter. 

Re-work of https://github.com/ArduPilot/ardupilot/pull/20497

Makes it much easier to add additional sensors in the future.

### CODE REVIEW

**Comment by IamPete1** [comment]
This costs about 100 bytes on vehicles. About 1K on periph because we gain the param conversion but don't save on the duplicate params because there is only a single airspeed instance.

**Review by tridge — DISMISSED** [review]
I'd prefer to see this without the rename of ARSP_TYPE to ARSPD1_TYPE

File: libraries/AP_Airspeed/AP_Airspeed.cpp [hunk 1]
```diff
@@ -78,103 +79,19 @@ extern const AP_HAL::HAL &hal;
...
 #define OPTIONS_DEFAULT AP_Airspeed::OptionsMask::ON_FAILURE_AHRS_WIND_MAX_DO_DISABLE | AP_Airspeed::OptionsMask::ON_FAILURE_AHRS_WIND_MAX_RECOVERY_DO_REENABLE | AP_Airspeed::OptionsMask::USE_EKF_CONSISTENCY
 
 // table of user settable parameters
 const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
 
-    // @Param: _TYPE
-    // @DisplayName: Airspeed type
-    // @Description: Type of airspeed sensor
-    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-MS5525 (0x76),5:I2C-MS5525 (0x77),6:I2C-SDP3X,7:I2C-DLVR-5in,8:DroneCAN,9:I2C-DLVR-10in,10:I2C-DLVR-20in,11:I2C-DLVR-30in,12:I2C-DLVR-60in,13:NMEA water speed,14:MSP,15:ASP5033,100:SITL
+    // @Param: _ENABLE
```
Comment (tridge) [inline libraries/AP_Airspeed/AP_Airspeed.cpp:87]: i don't think the ENABLE is useful on planes

**Comment by IamPete1** [comment]
Found some flash savings on Periph, so it should now come out saving a little. Vehicles will still cost 100 bytes or so. I need to re-test param conversion, periph and a vehicle.

**Comment by IamPete1** [comment]
Tested param conversion on both vehicles and periph.

**Review by tridge — APPROVED** [review]

**Pull request merged**